aptos-move/framework/aptos-framework/sources/configs/staking_config.move

```diff
@@ -146,7 +149,10 @@ module aptos_framework::staking_config {
             new_rewards_rate_denominator > 0,
             error::invalid_argument(EZERO_REWARDS_RATE_DENOMINATOR),
         );
-
+        assert!(
+            new_rewards_rate < 100,
+            error::invalid_argument(EINVALID_REWARDS_RATE),
+        );
         let staking_config = borrow_global_mut<StakingConfig>(@aptos_framework);
         staking_config.rewards_rate = new_rewards_rate;
```

Comment:
Same here, maybe this `new_rewards_rate < 100` can be a struct invariant.

Reply:
Yes, it's possible!

Reply:
Done.